dsl_parser/tasks.py

```diff
@@ -68,10 +68,9 @@ def prepare_deployment_plan(plan, inputs=None, **kwargs):
     """
     Prepare a plan for deployment
     """
-    plan = multi_instance.create_deployment_plan(plan)
```

Comment:
i dont understand this part of the changes. why was it made? what is it good for?